### ISSUE TITLE
None is an object not a string.

### ISSUE THREAD

**Issue by nwmoriarty** [qrefine/qrefine#16]
Another strange piece of code. Line 217 of qrefine/restraints.py is

 if self.qm_addon != 'None':

I believe it should be 

 if self.qm_addon is not None:

**Comment by hokru** [comment]
For me `self.qm_addon` is a string with the value 'None' by default, not an object. Testing for `is not None` is thus always true.

```
print type(self.qm_addon), self.qm_addon
<type 'str'> None
```


The error you saw might be because `qm_addon_method` was not specified (I'll add an error message), but let's resolve this issue first.


I assume it is a string because of the multi-choice. Using 'None' is perhaps counter intuitive.
```
  qm_addon = *None gcp dftd3
    .type = choice(multi=False)
  qm_addon_method = None
    .type = str
```

**Comment by mpwaller** [comment]
It is a special word in python, like null for java. It is therefore leading to confusion. Need to name it something else.

**Comment by hokru** [comment]
Ah great! Thanks @pafonine, that is the way to go. Will push the patch soon after some testing.

**Comment by mpwaller** [comment]
@hokru what is the status ?

**Comment by hokru** [comment]
Sorry I lost completely track of this. I pushed a fix now.

**Comment by nwmoriarty** [comment]
@hokru Can we close this?

**Comment by hokru** [comment]
resolved with a2fbf55d685d99b78ad047c513a9d5bdda94d220
